src/locale/messages.da.xlf

```diff
@@ -0,0 +1,187 @@
+          <context context-type="linenumber">12</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="1013806782562403538" datatype="html">
+        <source>rundown-header-context-menu-component.take.button</source>
+        <target>Take</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.html</context>
+          <context context-type="linenumber">13</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="2136327373490001416" datatype="html">
+        <source>rundown-component.take-next.button</source>
+        <target>Take næste</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown/rundown.component.html</context>
+          <context context-type="linenumber">4</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="410553181106118867" datatype="html">
+        <source>Loading the rundown...</source>
+        <target>Indlæser rundown...</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown/rundown.component.html</context>
+          <context context-type="linenumber">8</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="6602960182549626686" datatype="html">
+        <source>confirmation-dialog.close.label</source>
+        <target>Luk</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/shared/components/confirmation-dialog/confirmation-dialog.component.html</context>
+          <context context-type="linenumber">6</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="7515883357904500238" datatype="html">
+        <source>Are you sure?</source>
+        <target>Er du sikker?</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/shared/components/confirmation-dialog/confirmation-dialog.component.ts</context>
+          <context context-type="linenumber">20</context>
+        </context-group>
+        <note priority="1" from="description">confirmation-dialog.are-you-sure.title</note>
+      </trans-unit>
+      <trans-unit id="2807800733729323332" datatype="html">
+        <source>Yes</source>
+        <target>Ja</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/shared/components/confirmation-dialog/confirmation-dialog.component.ts</context>
+          <context context-type="linenumber">22</context>
+        </context-group>
+        <note priority="1" from="description">confirmation-dialog.yes.button</note>
+      </trans-unit>
+      <trans-unit id="2159130950882492111" datatype="html">
+        <source>Cancel</source>
+        <target>Annullere</target>
```

Comment:
`Anuller`

Reply:
`Annuller` or `Fortryd`.

Reply:
`Annuller`. `Fortryd` indicates you already did an action you want to undo i.e. do another action to remedy the first action.